javav2/example_code/cloudwatch/jsonWidgets.json

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
Is this file used? It seems like CloudDashboard.json and jsonWidgets.json both define dashboard widgets.

Reply:
One was suppose to have more widgets. I will fix